server/routes/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router = express.Router();
+
+const { createPizza , getPizzas} = require('../controllers/pizzaController');
+const { createOrder, getOrders, getOrderDetails } = require('../controllers/orderItemController');
+// const { getItems } = require('../controllers/orderController');
+
+
+router.post('/create/pizza',createPizza)

```

Comment:
Is this something you are used to?
How about using post, get, and put  as the explanatory message of the current endpoint?

LIke pizzas/, then you just call get and post on them. Nothing more